app/models/communitarian/resolution.rb

```diff
@@ -51,5 +46,10 @@ def close_by_vote?(post)
     def resolution?(post)
       post.topic.custom_fields["is_resolution"] && post.polls.exists?
     end
+
+    def has_recent_polls?(topic_id)
```

Comment:
This check will not work if user will create a regular poll inside resolution topic. Right now in master branch we don't have different between regular poll and resolution poll inside topic. I've added this difference in another PR(https://github.com/fs/communitarian-discourse-plugin/pull/21/files#diff-b61ae67bc6a57a3decb8d15b832077f4R66), but I think that we don't this check since now reopen_resolution job always will have run_at in the future:
```
Jobs.enqueue_at(resolution_schedule.next_reopen_time, :reopen_resolution, job_args)
```

Reply:
I've added this check to throttle resolutions creation because __somehow__ too many polls were created at the same time.

Reply:
I think there should be no polls added manually in resolutions replies.